recipes/PySP2/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "pysp2" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pysp2-{{ version }}.tar.gz
+  sha256: 234400bc9205bbe47659b407062a8885b309d19fa93ce2651b821af67d7cdf71
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python
+    - scipy
+    - xarray
+    - xlrd
+    - act-atmos
+    - numba
+
+test:
+  imports:
+    - pysp2
+  commands:
+    - pytest 
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests/*.py
+
+about:
+  home: https://pypi.org/project/pysp2/
+  summary: Py-SP2
+  dev_url: https://github.com/rcjackson/PySP2
```

Should this be this?
```suggestion
  dev_url: https://github.com/ARM-DOE/PySP2
```